packages/cms-lib/api/results.js

```diff
@@ -3,12 +3,13 @@ const http = require('../http');
 const RESULTS_API_PATH = 'cms/v3/functions/results';
 
 async function getFunctionLogs(portalId, functionId, query = {}) {
-  const { limit = 5 } = query;
+  const { limit = 5, after } = query;
 
   return http.get(portalId, {
     uri: `${RESULTS_API_PATH}/${functionId}`,
     query: {
```

Comment:
I'd expect `query` to support all the options, which makes me wonder if it should be passed through.